src/mlpack/core/tree/rectangle_tree/hilbert_r_tree_split.hpp

```diff
@@ -1 +1,86 @@
+/**
+ * @file hilbert_r_tree_split.hpp
+ * @author Mikhail Lozhnikov
+ *
+ * Defintion of the HilbertRTreeSplit class, a class that splits the nodes of an R
+ * tree, starting at a leaf node and moving upwards if necessary.
+ */
+#ifndef MLPACK_CORE_TREE_RECTANGLE_TREE_HILBERT_R_TREE_SPLIT_HPP
+#define MLPACK_CORE_TREE_RECTANGLE_TREE_HILBERT_R_TREE_SPLIT_HPP
+
+#include <mlpack/core.hpp>
+
+namespace mlpack {
+namespace tree /** Trees and tree-building procedures. */ {
+
+/**
+ * The order of the splitting policy. The Hilbert R tree splits a node
+ * on overflow, turnung splitOrder node to (splitOrder+1) nodes.
+ */
+constexpr int splitOrder = 2;
```

Comment:
I don't know if it's good to just leave `splitOrder` open in the `mlpack::tree` namespace, maybe we should put it inside `HilbertRTreeSplit` as a static member---or alternately make it a template parameter.  Whatever you want to do is up to you, I'd just like to get it out of `mlpack::tree` since there is a lot of other stuff in that namespace too.